Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -347,6 +367,9 @@ void EditorActionsHandler::OnActionRegistrationHook()
                 UpdateRecentFileActions();
             }
         );
+
+        // This action is only accessible outside of Component Modes
+        m_actionManagerInterface->AssignModeToAction(AzToolsFramework::DefaultActionContextModeIdentifier, actionIdentifier);
```

Comment:
question: Might it be possible to make this the default and people opt out of this behavior as opposed to the other way round? I guess it depends on if there are more things that should be accessed outside component modes or inside (my thought was always there's more things that are only accessible outside, but I could be wrong, I'd be interested to hear your thoughts).

Reply:
Thanks for the feedback - totally fine to leave as-is and we can always revisit if needed. There are upsides and downsides to either approach and this can be useful in a number of situations. We were overly cautious with Component Mode as we had seen how bad things could get in the old system with lots of competing systems left 'on' inadvertently that would compete for inputs and logic and cause all kinds of weird bugs. This meant we maybe turned off more that was 'technically' allowed but it did mean Component Mode was more isolated by default. The drawback was some operations that might have been useful at times were off limits (our justification was as it was quick and easy to leave Component Mode it wasn't a big deal) but I can see it being an irritation for customers at times too.